Ingestion Controls: Agent Level configuration modal (#14969)
DataDog/documentation: content/en/tracing/trace_pipeline/ingestion_controls.md

The Ingestion Control page provides visibility at the Agent and tracing libraries level into the ingestion configuration of your applications and services. From the [ingestion control configuration page][2], you can:
- Gain visibility on your service-level ingestion configuration and adjust trace sampling rates for high throughput services.
- Understand which ingestion mechanisms are responsible for sampling most of your traces.
- Investigate and act on potential ingestion configuration issues, such as limited CPU or RAM resources for the Agent.

{{< img src="tracing/trace_indexing_and_ingestion/ingestion_controls_page.png" style="width:100%;" alt="Ingestion Control Page Overview" >}}

All metrics used in the page are based on live traffic data of the **past 1 hour**. Any Agent or library configuration change is reflected in the page.

## Summary across all environments

Get an overview of the total ingested data over the past hour, and an estimation of your monthly usage against your monthly allocation, calculated with the active APM intrastructure (hosts, Fargate tasks, and serverless functions).

> If the monthly usage is under `100%`, the projected ingested data fits in your [monthly allotment][3]. A quota over `100%` means that the monthly ingestion volume is projected to be over your monthly allocation.

Author: let's use monthly allotment everywhere
and ingested data

> If the monthly usage is under `100%`, the projected ingested data fits in your [monthly allotment][3]. A quota over `100%` means that the monthly ingestion volume is projected to be over your monthly allocation.

Reviewer: ```suggestion
If the monthly usage is under `100%`, the projected ingested data fits in your [monthly allotment][3]. A monthly usage value over `100%` means that the monthly ingested data is projected to be over your monthly allotment.
```

For further investigations, use the [APM Trace - Estimated Usage Dashboard][12], which provides global ingestion information as well as breakdown graphs by `service`, `env` and `ingestion reason`.

### Configure the service ingestion rate

Click **Manage Ingestion Rate** to get recommendations on how to configure your service ingestion rate.

{{< img src="tracing/trace_indexing_and_ingestion/service_ingestion_rate_config.png" style="width:100%;" alt="Change the Service Ingestion Rate" >}}

To specify that a specific percentage of a service's traffic should be sent, add an environment variable or a generated code snippet to your tracer configuration for that service.

1. Select the service you want to change the ingested span percent for.
2. Choose the service language.
3. Choose the desired ingestion percentage.
4. Apply the appropriate configuration generated from these choices to the indicated service and redeploy the service.

> 5. Confirm on the Ingestion Control Page that your new percentage has been applied: the Traffic Breakdown column should surface the sampling rate applied. The ingestion reason for the service should also be `ingestion_reason:rule`.

Reviewer: ```suggestion
5. Confirm on the Ingestion Control Page that your new percentage has been applied by looking at the Traffic Breakdown column, which surfaces the sampling rate applied. The ingestion reason for the service is shown as `ingestion_reason:rule`.
```
